yarn-project/prover-node/src/prover-coordination/aztec-node-prover-coordination.ts

```diff
@@ -0,0 +1,14 @@
+import type { AztecNode, EpochProofQuote, ProverCoordination, Tx, TxHash } from '@aztec/circuit-types';
+
+/** Implements ProverCoordinator by wrapping an Aztec node */
+export class AztecNodeProverCoordination implements ProverCoordination {
```

Comment:
Curious: why do we need this wrapper? Can't we use an AztecNode directly?

Reply:
Agreed. If we add `implements ProverCoordination` to the Aztec Node then we should not need this class.

Added https://github.com/AztecProtocol/aztec-packages/issues/8852